run.cmd

```diff
@@ -9,11 +9,12 @@ if exist %_VSWHERE% (
 )
 if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
 if not exist "%_VSCOMNTOOLS%" (
-  echo Error: Visual Studio 2015 or 2017 required.
-  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
-  exit /b 1
+    echo %__MsgPrefix%Error: Visual Studio 2015 or 2017 required.
```

Comment:
We don't set __MsgPrefix anywhere in this repo that I could find from a quick search so can undo this part of the change.